### PR TITLE
De-duplication for group calls

### PR DESCRIPTION
**Pull request by SimonBrandner**
---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

File: src/webrtc/groupCallEventHandler.ts [hunk 1]
```diff
@@ -50,14 +50,14 @@ export class GroupCallEventHandler {
 
         const groupCallId = event.getStateKey();
 
-        const callType = content.type;
+        const callType = content["m.type"];
```
Comment (robertlong) [inline src/webrtc/groupCallEventHandler.ts:53]: Yup, thanks for changing these. Was in a rush to push things out last night so you could continue working.

File: src/webrtc/call.ts [hunk 2]
```diff
@@ -570,40 +570,19 @@ export class MatrixCall extends EventEmitter {
...
-                    `Adding track (` +
-                    `id="${track.id}", ` +
-                    `kind="${track.kind}", ` +
-                    `streamId="${stream.id}", ` +
-                    `streamPurpose="${purpose}"` +
-                    `) to peer connection`,
-                );
-                senderArray.push(this.peerConn.addTrack(track, stream));
-            }
+            this.pushLocalFeed(callFeed, addToPeerConnection);
```
Comment (robertlong) [inline src/webrtc/call.ts:584]: Much cleaner, thanks!

Comment (SimonBrandner) [reply src/webrtc/call.ts:583]: Sorry, for not making it more clear on the first go :P

File: src/webrtc/call.ts [hunk 3]
```diff
@@ -581,7 +581,6 @@ export class MatrixCall extends EventEmitter {
         } else {
             const callFeed = new CallFeed(stream, userId, purpose, this.client, this.roomId, false, false);
             this.pushLocalFeed(callFeed, addToPeerConnection);
-            this.emit(CallEvent.FeedsChanged, this.feeds);
```
Comment (robertlong) [inline src/webrtc/call.ts:592]: Thanks for fixing this

**Review by robertlong — APPROVED** [review]
Looks good! Merging into my branch. Thanks again!

File: src/webrtc/call.ts (hunk 3 above)
Comment (SimonBrandner) [reply src/webrtc/call.ts:592]: np!

**Pull request merged**